src/network/protocol/messages/lib.zig

```diff
@@ -3,26 +3,30 @@ pub const VersionMessage = @import("version.zig").VersionMessage;
 pub const VerackMessage = @import("verack.zig").VerackMessage;
 pub const MempoolMessage = @import("mempool.zig").MempoolMessage;
 pub const GetaddrMessage = @import("getaddr.zig").GetaddrMessage;
+pub const GetblocksMessage = @import("getblocks.zig").GetblocksMessage;
 
 pub const MessageTypes = enum {
     Version,
     Verack,
     Mempool,
     Getaddr,
+    Getblocks,
 };
 
 pub const Message = union(MessageTypes) {
     Version: VersionMessage,
     Verack: VerackMessage,
     Mempool: MempoolMessage,
     Getaddr: GetaddrMessage,
+    Getblocks: GetblocksMessage,
 
     pub fn deinit(self: Message, allocator: std.mem.Allocator) void {
         switch (self) {
             .Version => |m| m.deinit(allocator),
             .Verack => {},
             .Mempool => {},
             .Getaddr => {},
+            .Getblocks => {},
```

Comment:
you need a deinit method to deallocate the `header_hashes` the struct contains.

Reply:
ok